cbcontainers/state/hardening/objects/enforcer_deployment.go

```diff
@@ -19,8 +19,9 @@ const (
 	DesiredContainerPortName  = "https"
 	DesiredContainerPortValue = 443
 
-	DesiredTlsSecretVolumeName      = "cert"
-	DesiredTlsSecretVolumeMountPath = "/etc/octarine-certificates"
+	DesiredTlsSecretVolumeName          = "cert"
+	DesiredTlsSecretVolumeMountPath     = "/etc/octarine-certificates"
+	DesiredTlsSecretVolumeMountReadOnly = true
```

Comment:
these should be `DesiredTLS...`, but you don't have to change them now, since 2 of them are old